### PR TITLE
✨ Converts encrypted SwG document keys to Base64 URL format

### PR DESCRIPTION
**Pull request by ChrisAntaki**
This PR converts encrypted document keys to Base64 URL format before sending them as URL parameters

### CODE REVIEW

File: extensions/amp-subscriptions/0.1/test/test-crypto-handler.js [hunk 1]
```diff
@@ -44,15 +44,14 @@ describes.realWin(
     // eslint-disable-next-line max-len
     const decryptedContent =
       "\n              This is section is top secret.\n              You should only be able to read this if you have the correct permissions.\n              If you don't have the correct permissions, you shouldn't be able to read this section at all.\n            ";
-    // eslint-disable-next-line max-len
-    const encryptedKey =
-      "ENCRYPT({'AccessRequirements': ['googleAccessRequirements:123'], 'Key':'mSfq5tRx5omXoOX20Oqq8g=='})";
+    const encryptedKeyBase64 = 'Oik/+==';
```
Comment (elijahsoria) [inline extensions/amp-subscriptions/0.1/test/test-crypto-handler.js]: Please add a comment saying what this is in plaintext.

Comment (ChrisAntaki) [reply extensions/amp-subscriptions/0.1/test/test-crypto-handler.js]: Added 👍

**Comment by ChrisAntaki** [comment]
Just to clarify, the key is being stored in Base 64 format at rest, and is just translated to Base 64 URL format in the `getEncryptedDocumentKey` function. 🙂

File: extensions/amp-subscriptions/0.1/crypto-handler.js [hunk 2]
```diff
@@ -64,17 +64,25 @@ export class CryptoHandler {
...
+   * Returns `null` if the key wasn't found.
    * @param {string} serviceId Who you want to decrypt the key.
    *                           For example: 'google.com'
    * @return {?string}
    */
   getEncryptedDocumentKey(serviceId) {
     // Doing this for testing.
     const encryptedKeys = this.getEncryptedKeys();
-    if (!encryptedKeys) {
+    const key = encryptedKeys && encryptedKeys[serviceId];
```
Comment (dvoytenko) [inline extensions/amp-subscriptions/0.1/crypto-handler.js]: Wait. Didn't we say we won't do this? And just use a normal url-encoding instead?

Comment (ChrisAntaki) [reply extensions/amp-subscriptions/0.1/crypto-handler.js]: Thanks for clarifying! I realize this PR is redundant, I'll close

**Pull request closed without merging**